src/storage/storage_info.cpp

```diff
@@ -2,6 +2,6 @@
 
 namespace duckdb {
 
-const uint64_t VERSION_NUMBER = 22;
+const uint64_t VERSION_NUMBER = 25;
```

Comment:
Could we get some tests for the UUID type, e.g. using it in tables, queries, (persistent) storage, UUID casts, etc?

Reply:
I had added some tests, do you think we should add more?

Reply:
Could we get some tests with:

* Comparisons between UUIDs (`=`, `<>`, `>=`, etc)
* `GROUP BY` on UUID
* A few more incorrect string -> UUID conversion (e.g. empty string `SELECT ''::UUID`, too few characters `SELECT 'aaaa'::UUID`, illegal characters `SELECT 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380Z11'::UUID`)